### ISSUE TITLE
Installation: Add more Helm variables

### ISSUE THREAD

**Issue by elliotxx** [KusionStack/karpor#347]
### What would you like to be added?

Add more Helm variables, example:
- `Resources` configuration of karpor-server, karpor-sync. Can adapt to some small development machines. #518 **[Completed]**
- Specific `image version` of karpor-server, karpor-sync #519 **[Completed]**
- Specific `replicas` of karpor-server, karpor-sync **[Completed]**
- Support flexible config `storage` of etcd, es, such as PVC, hostPath etc. #522 #548 #550 
- Support custom fields of service. #548 
- ......

### Why is this needed?

Add more Helm variables to improve installation flexibility and adapt to more installation scenarios.

**Comment by elliotxx** [comment]
@T1-leiyang Thanks for your enthusiasm! 🔥 But I have already started working on this issue. Maybe I can wait for you to complete the #461 , let's complete other tasks of this issue together.

**Comment by elliotxx** [comment]
@T1-leiyang Hi, I have preliminary optimized the Helm Chart in the following work. Currently, these [variables](https://www.kusionstack.io/karpor/next/getting-started/installation#chart-parameters) are supported, but they may not be sufficient. If you are willing, you can add more variables on this basis:
- https://github.com/KusionStack/charts/pull/36
- https://github.com/KusionStack/charts/pull/37
- https://github.com/KusionStack/charts/pull/38

**Comment by danielstankw** [comment]
@elliotxx if needed I can help improve the helm chart

**Comment by elliotxx** [comment]
> @elliotxx if needed I can help improve the helm chart

@danielstankw Yeah, I am now debugging another feature #497 . The optimization (PVC, StorageClass, Service ...) of Helm Chart originally planned to start later this week. But if you can help optimize it, it will be great!

**Comment by elliotxx** [comment]
@danielstankw Oh, sorry, the chart repo is here: https://github.com/KusionStack/charts/tree/master/charts/karpor
